### PR TITLE
Update SwiftUI and UIKit tutorials

### PR DESCRIPTION
**Pull request by testableapple**

### CODE REVIEW

File: docusaurus/docs/iOS/02-tutorials/02-audio-room-uikit.mdx [hunk 1]
```diff
@@ -139,13 +139,20 @@ Joining an audio room requires microphone access, you need to request permission
...
     private let call: Call
     private var callCreated: Bool = false {
         didSet {
-            var content = contentView.content
-            content.callCreated = true
-            contentView.content = content
+            contentView.content = .init(
+                callCreated: true,
+                callId: call.callId,
+                title: "",
```
Comment (testableapple) [inline docusaurus/docs/iOS/02-tutorials/02-audio-room-uikit.mdx:153]: @ipavlidakis, what title should be used here?

Comment (ipavlidakis) [reply docusaurus/docs/iOS/02-tutorials/02-audio-room-uikit.mdx:153]: I think empty is fine. This title will be later replaced from the descriptionView which will use a Combine publisher to access the title and the description.

**Review by ipavlidakis — APPROVED** [review]

File: docusaurus/docs/iOS/02-tutorials/02-audio-room-uikit.mdx [hunk 2]
```diff
@@ -633,14 +650,18 @@ final class MicButtonView: UIButton {
     @available(*, unavailable)
     required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
 
-    init() {
+    init(microphone: MicrophoneManager) {
         super.init(frame: .zero)
-        setImage(.init(systemName: "mic.circle"), for: .normal)
+        setImage(.init(systemName: microphone.status == .enabled ? "mic.circle" : "mic.slash.circle"), for: .normal)
```
Comment (ipavlidakis) [inline docusaurus/docs/iOS/02-tutorials/02-audio-room-uikit.mdx:655]: We shouldn't change this one as it's being handled in the next iteration of the microphone manager

**Review by ipavlidakis — CHANGES_REQUESTED** [review]

**Review by ipavlidakis — COMMENTED** [review]
Remove unnecessary PR review status

**Review by ipavlidakis — APPROVED** [review]

**Pull request merged**